package.json

```diff
@@ -11,10 +11,10 @@
     "crypto"
   ],
   "main": "./bcrypt",
-  "version": "2.0.1",
+  "version": "3.0.0",
```

Comment:
Please do not make this increment. We will release with a CHANGELOG and the increment is done as the part of it.